cmd/cri-containerd/cri_containerd.go

```diff
@@ -68,3 +67,19 @@ func main() {
 		glog.Exitf("Failed to run cri-containerd grpc server: %v", err)
 	}
 }
+
+func validateConfig(o *options.CRIContainerdOptions) {
```

Comment:
`validateConfig` to `validateSecurityConfig`?  Because it only contains the config related to security.

Reply:
It may validate other configurations in the future I think, if there are others need to be validated. :)

Reply:
Makes sense